docs/patterns/single-new-eks-observability-accelerators/single-new-eks-container-logs-opensource-observability.md

```diff
@@ -0,0 +1,85 @@
+# Single Cluster Open Source Observability - NGINX Monitoring
+
+## Objective
+
+This pattern demonstrates how to use the _New EKS Cluster Open Source Observability Accelerator_ forward container logs to cloud watch using ADOT containers log collector.
```

Comment:
Could you please add some narrative and also refernce the launch blog or announcement doc which we released for reInvent for this

Reply:
Updated the documentation and added a link the announcement doc